### ISSUE TITLE
Some questions about Mincer exact

### ISSUE THREAD

**Issue by vsht** [vermaseren/form#385]
I apologize if my questions are too naive, but I'm not a Mincer expert (when
I started using IBPs FIRE was already there :) ) and I'd really like to understand
certain things in a better way.

1. Where can I see which master integrals are implemented to which order in ep?  For example,
it seems that the product of 3 1-loop bubbles is available only up to ep^4, even if one asks for more
orders. Of course, it might be that I'm using the code in a wrong way:

```
off stat;
#include minceex.h
.global
#define MSBAR "1"


* massless bubble ^3 , LA topology, fig. 7 in the
* mincer-form paper, remove p7,p8
L myInt = 1/p1.p1/p2.p2/p3.p3/p4.p4/p5.p5/p6.p6;


#call integral(la,0)
#call subvalues
* expand up to ep^6
#call expansion(8)
id Q.Q^s? = 1;
Print +f;
.sort
Format mathematica;
#write  <Bubble3L.m> "(%E)",myInt

.end 
```

2. Could some kindly point me to a reference that gives  the explicit normalization factor for 
Mincer integrals? Empirically, I found that if one wants to convert Mincer output (in the MSbar scheme),
to the "standard" multiloop normalization `(Exp[ep EulerGamma]/(I Pi^(d/2)))^L`, I need to multiply
it by `1 - (Pi^2 ep^2)/4 + (Pi^4 ep^4)/32 - (Pi^6 ep^6)/384 + ...` (not 100% sure about the ep^6-term though).
Is this correct?

Cheers,
Vladyslav

**Comment by apik** [comment]
You can use MINCER for reduction only, and in the end substitute master integrals in any chosen normalization. Here is an example for chain of self-energies:
```
-
#include minceex.h
* master integrals
S [l1(1,1)];
S [t1(1,1,1,1,0)],[t1(1,0,1,0,1)];
S [no(1,1,1,1,1,1,1,1)],[la(1,1,1,1,1,1,0,0)],[la(1,0,1,0,1,0,1,1)],
[la(1,0,1,1,1,0,1,0)],[la(1,0,1,1,0,1,1,1)],[la(0,0,1,0,0,1,1,1)];


L se3 = 1/<p1.p1>/.../<p6.p6>;

#define TOP "la"
#call momsubs(`TOP')
#call integral(`TOP',1)

* 3-loop
#if(`TOP'==la)||(`TOP'==be)||(`TOP'==no)
id BNO = [no(1,1,1,1,1,1,1,1)]*Q.Q^2*rat(1, 1);
id GC0*BT1 = rat(3*ep-1,-2*ep^3)*[la(1,0,1,1,0,1,1,1)] - [la(0,0,1,0,0,1,1,1)]*rat( - 60*ep^3 + 97*ep^2 - 47*ep + 6,ep^5)/Q.Q^2;
id GC0^3 = [la(1,1,1,1,1,1,0,0)]*rat(1, 1);
id GC0^2*GC1 = ([la(1,0,1,1,1,0,1,0)]*rat(2*(-2 + 3*ep), ep))/Q.Q;
id GC0^2*GC2 = ([la(1,0,1,0,1,0,1,1)]*rat(3*(-1 + 2*ep), ep))/Q.Q;
id GC0*GC1*GC2 = ([la(0,0,1,0,0,1,1,1)]*rat(6*(-2 + 3*ep)*(-3 + 4*ep), ep^2))/Q.Q^2;
* 2-loop
#elseif(`TOP'==t1)
id GC0*GC1 = [t1(1,0,1,0,1)]*rat(6*ep - 4,ep)/Q.Q;
id GC0^2 = [t1(1,1,1,1,0)];
* 1-loop
#elseif(`TOP'==l1)
id GC0 = [l1(1,1)];
#else
#message Unknown topology `TOP'
#endif

Print+s;
.end
```

**Comment by vsht** [comment]
Thanks to  the two of you for the explanations!

@apik Actually, I wanted to do the opposite: Upon finishing the reduction with FIRE I noticed that some
of my masters where "Mincer" integrals, so it seemed a natural thing to get the analytic results form Mincer
instead of looking them up in the literature or recalculating with HyperInt.

The normalization was not so clear to me, but then I simply fixed it by comparing the 1L bubble cubed with
the known literature result. However, when checking the so-obtained results numerically (for real 3L integrals), 
I noticed that for some integrals O(ep^3) pieces were not correct, while O(ep^2) pieces matched perfectly. I guess this
is because such high orders are simply not implemented, as explained by Jos. On the other hand, in all
cases Mincer gave me analytic results up to O(ep^4), which was the reason for my question regarding
the maximal O(ep)-order implemented for each master available in Mincer.

I'll then switch to FORCER for higher orders in the ep-expansion.
